package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "carousel",
+  "version": "0.1.0",
+  "description": "Displays images in a carousel either static or dynamic images",
+  "author": "Akileng Isaac <isaac.akileng@flockofbirds.org>",
+  "copyright": "Flock of Birds International BV",
+  "main": "Gruntfile.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mendixlabs/carousel"
+  },
+  "license": "Apache License 2.0",
+  "bugs": {
+    "url": "https://github.com/mendixlabs/carousel/issues"
+  },
+  "devDependencies": {
+    "grunt": "1.0.1",
+    "grunt-contrib-clean": "^1.0.0",
+    "grunt-contrib-compress": "^1.2.0",
+    "grunt-contrib-copy": "^1.0.0",
+    "grunt-contrib-csslint": "^1.0.0",
+    "grunt-contrib-watch": "^1.0.0",
+    "grunt-newer": "^1.1.1",
+    "grunt-xsltproc": "^0.6.1",
+    "load-grunt-tasks": "^3.1.0",
+    "node-base64-image": "^0.1.0",
+    "node-mendix-modeler-path": "https://github.com/JelteMX/node-mendix-modeler-path/archive/v1.0.0.tar.gz",
+    "semver": "^5.1.0",
+    "shelljs": "^0.7.0",
+    "tslint-eslint-rules": "^2.1.0",
+    "typescript": "^2.0.3",
+    "xml2js": "^0.4.15"
+  },
+  "dependencies": {
+    "grunt-xsltproc": "^0.6.1",
```

Comment:
This seems to be a dev dependency, if I'm not mistaken this is used to generate `.d.ts` files from the property xml file right?